### ISSUE TITLE
Files need to be converted to unix line ending format

### ISSUE THREAD

**Issue by EdFuentetaja** [Microtelecom/libperseus-sdr#1]
First of all, thanks for sharing this and congratulations for the excellent Perseus hardware and software.

I tried to compile this code on Fedora and I noticed that the ASCII files need to be converted first to the UNIX line ending format. You can use a tool like dos2unix,

**Comment by EdFuentetaja** [comment]
I tried to compile this code on Windows as well. Several issues here:
- After struggling with MinGW I had to give up. I suspect there is an error on the configure.ac file but I'm not an expert on this toolchain.
- I finally used Visual Studio. Some sources need some modifications but the code compiles. Here I have an issue on execution. The test application stalls in the perseus_fx2_fpga_config_sr. This takes forever. I suspect the problem is with the libusb Windows implementation. On Linux the test application works fine.

I hope this report is helpful. Thanks again for the excellent job.

**Comment by EdFuentetaja** [comment]
I tried to compile this code on Windows as well. Several issues here:

After struggling with MinGW I had to give up. I suspect there is an error on the configure.ac file but I'm not an expert on this toolchain.
I finally used Visual Studio. Some sources need some modifications but the code compiles. Here I have an issue on execution. The test application stalls in the perseus_fx2_fpga_config_sr. This takes forever. I suspect the problem is with the libusb Windows implementation. On Linux the test application works fine.
I hope this report is helpful. Thanks again for the excellent job.
